src/diffusers/loaders.py

```diff
@@ -916,6 +927,12 @@ def load_textual_inversion(
         for token_id, embedding in token_ids_and_embeddings:
             self.text_encoder.get_input_embeddings().weight.data[token_id] = embedding
 
+        # offload back
+        if is_model_cpu_offload:
+            self.enable_model_cpu_offload()
+        else:
+            self.enable_sequential_cpu_offload()
```

Comment:
```suggestion
        elif is_model_sequantial_cpu_offload:
            self.enable_sequential_cpu_offload()
```
I don't think we should **always** enable either `model_cpu` or `sequential_cpu` offload. We should only enable  it if sequential cpu offload was enabled before